src/components/whatsAppTemplates/FormTabs.vue

```diff
@@ -366,14 +366,17 @@
           };
 
           await this.createTemplate({ appUuid, payload: templatePayload });
+
           if (this.errorCreateTemplate) {
-            this.callErrorModal({ text: this.$t('WhatsApp.templates.error.create_template') });
+            const errorText =
+              this.errorCreateTemplate?.error_user_msg ||
+              this.$t('WhatsApp.templates.error.create_translation');
+            this.callErrorModal({ text: errorText });
             this.loadingSave = false;
             return;
           }
 
-          currentTemplateUuid = this.createdTemplateData.uuid;
-          this.loadingSave = false;
```

Comment:
why `loadingSave` statement was removed? it is still being setted as true at the start of the method